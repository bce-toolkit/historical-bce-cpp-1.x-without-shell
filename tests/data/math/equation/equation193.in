6 5
1 0 -2 0 0 0
3 0 -4 0 -2 0
0 1 0 -1 0 0
0 1 0 -1 0 0
0 1 0 0 -1 0
# NH3+NaClO=N2H4+NaCl+H2O
