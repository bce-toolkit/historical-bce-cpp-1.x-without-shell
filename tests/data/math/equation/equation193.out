2Xa
Xa
Xa
Xa
Xa
